### PR TITLE
Build responsive UI with constraintlayout

### PR DESCRIPTION
**Pull request by corinaorasan**
#### Description
Add more TextViews and three buttons with three different colors and add click handler to color different  three TextViews. The focus is on constraint layout.
#### [Task] 
https://classroom.udacity.com/courses/ud9012/lessons/4f6d781c-3803-4cb9-b08b-8b5bcc318d1c/concepts/152b8298-d6b3-4b29-8edd-1ce13c7c96a6
## Steps to Test or Reproduce
1.Start the app.
2.Tap the screen and press the buttons.
## Risks

### CODE REVIEW

File: app/src/main/java/com/example/colormyviews/MainActivity.kt [hunk 1]
```diff
@@ -1,11 +1,47 @@
...
+        box_two_text -> view.setBackgroundColor(Color.GRAY)
+        box_three_text -> view.setBackgroundResource(android.R.color.holo_green_light)
+        box_four_text -> view.setBackgroundResource(android.R.color.holo_green_dark)
+        box_five_text -> view.setBackgroundResource(android.R.color.holo_green_light)
+        red_button -> box_three_text.setBackgroundResource(R.color.my_red_color)
+        yellow_button -> box_four_text.setBackgroundResource(R.color.my_yellow_color)
+        green_button -> box_five_text.setBackgroundResource(R.color.my_green_color)
+        else -> view.setBackgroundColor(Color.LTGRAY)
+    }
+
```
Comment (deniscoman) [inline app/src/main/java/com/example/colormyviews/MainActivity.kt]: We should remove extra line spaces :)

File: app/src/main/java/com/example/colormyviews/MainActivity.kt [hunk 2]
```diff
@@ -1,11 +1,47 @@
...
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        setListeners()
     }
+
+    private fun setListeners() {
+
```
Comment (deniscoman) [inline app/src/main/java/com/example/colormyviews/MainActivity.kt]: Remove extra line

**Review by deniscoman — APPROVED** [review]

**Review by deniscoman — CHANGES_REQUESTED** [review]
Can you add another pink button and textView?

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -100,6 +100,22 @@
         app:layout_constraintTop_toBottomOf="@+id/box_two_text"
         app:layout_constraintVertical_bias="0.277" />
 
+    <TextView
+        android:id="@+id/box_six_text"
+        style="@style/WhiteBox"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
```
Comment (AlexandraDamaschin) [inline app/src/main/res/layout/activity_main.xml]: We should refactor this and use the from the dimensions file

**Review by AlexandraDamaschin — APPROVED** [review]
Does this improve software quality? Yes

**Pull request merged**